packages/babel-plugin-ember-test-metadata/get-relative-paths.js

```diff
@@ -14,7 +14,7 @@ function _getRelativeProjectPath(pathSegments, projectDir, projectRoot) {
 
 function _getRelativePathForClassic(filePath, packageName, projectRoot) {
   const projectDir = _getNormalizedPackageDir(packageName);
-  const pathSegments = filePath.split(sep);
+  const pathSegments = filePath.split(sep).filter(item => item !== 'dummy')
```

Comment:
Minor nitpick for formatting and variable naming:
```suggestion
  const pathSegments = filePath.split(sep).filter((segment) => segment !== 'dummy')
```